src/main/java/net/openhft/chronicle/wire/Marshallable.java

```diff
@@ -164,8 +164,17 @@ default <T> T deepCopy() throws InvalidMarshallableException {
         return (T) Wires.deepCopy(this);
     }
 
-    default <T extends Marshallable> T copyTo(@NotNull T t) throws InvalidMarshallableException {
-        return Wires.copyTo(this, t);
+    /**
+     * Copy fields from this to dest by marshalling out and then in. Allows copying of fields by name
+     * even if there is no type relationship between this and dest
+     *
+     * @param dest destination
+     * @return t
+     * @param <T> destination type
+     * @throws InvalidMarshallableException
```

Comment:
It shouldn't be throwing this exception after this change

Reply:
excellent - thanks - removed